gateways/node_api_gateway.py

```diff
@@ -16,6 +16,9 @@
     HathorCoreClient,
 )
 
+# The default lambda timeout for Node API Lambdas is set to 6 seconds.
```

Comment:
I suggest informing on this comment that the timeouts set here will be shorter than the lambdas timeout itself.
Without navigating to the issue that originated this PR, it's not obvious why there is this difference.